readthedocs/rtd_tests/tests/test_middleware.py

```diff
@@ -32,6 +32,7 @@ def setUp(self):
     def test_failey_cname(self):
         self.assertFalse(Domain.objects.filter(domain='my.host.com').exists())
         request = self.factory.get(self.url, HTTP_HOST='my.host.com')
+        request.user = self.owner
```

Comment:
Would setting `request.user = AnonymousUser()` be closer to the original test case?

Reply:
Yeah, I think so, I retreated from using it bc of another failure but in the end was unrelated, I'll test again.

Reply:
AnonymousUser works!